sdv/sequential/par.py

```diff
@@ -223,9 +224,13 @@ def add_constraints(self, constraints):
         constraint_cols = []
         for constraint in constraints:
             if isinstance(constraint, ProgrammableConstraint):
-                raise SynthesizerInputError(
-                    'The PARSynthesizer cannot accommodate with programmable constraints.'
-                )
+                if not isinstance(constraint, SingleTableProgrammableConstraint):
+                    raise SynthesizerInputError(
+                        'The PARSynthesizer only supports SingleTableProgrammableConstraint '
```

Comment:
Is this line getting hit?  We check for a different error message in the reject multi test case.

Reply:
What do you mean by "reject multi test case"?

Reply:
`test_reject_multi_table_programmable_constraint`

Reply:
Updated